contracts/src/v0.8/automation/chains/ChainModuleBase.sol

```diff
@@ -18,7 +18,7 @@ contract ChainModuleBase is IChainModule {
     return blockhash(n);
   }
 
-  function getCurrentL1Fee() external view virtual returns (uint256) {
+  function getCurrentL1Fee(uint256) external view virtual returns (uint256) {
```

Comment:
this whole contract could benefit from some natspec. It's not apparent what this parameter is for 🤔

Reply:
added natspec for common functions in `IChainModule` and
added natspec for functions unique to OptimismModuleV2 and ScrollModule